scripts/auto-rebase/rebase.sh

```diff
@@ -43,6 +42,159 @@ title() {
     echo -e "\E[34m$1\E[00m";
 }
 
+# LVMS is not integrated into the ocp release image, so the work flow does not fit with core component rebase.  LVMS'
+# operator bundle is the authoritative source for manifest and image digests.
+download_lvms_operator_bundle(){
+    title 'downloading LVMS operator bundles'
+    declare -A images=(["amd64"]="${1}" ["arm64"]="${2:-}")
+    local LVMS_STAGING="${STAGING_DIR}/lvms"
+    for arch in "${!images[@]}"; do
+        # TODO special case arm64 builds of topolvm since they are not available in 4.12
```

Comment:
ARM images are available as part of the same manifest. 
See https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel8/63972de3adcb55263891b983?architecture=arm64&container-tabs=gti

Reply:
Ah, yes.  This is stale, will remove.